helpers/test/controller.go

```diff
@@ -0,0 +1,75 @@
+package test_helpers
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+
+	"github.com/beego/beego/v2/server/web"
+	"github.com/onsi/ginkgo"
+)
+
+// MakeRequest make a HTTP request and return response
+func MakeRequest(method string, url string, body io.Reader) *httptest.ResponseRecorder {
```

Comment:
Looks great 👍🏻 But just a small comment.
Request header is also unstable value. Imagine if you need to make a request with and without authorization header.

I recommend to send headers as a parameter to this function
```go
func MakeRequest(method string, url string, body io.Reader, headers http.Header) *httptest.ResponseRecorder {
  request := HTTPRequest(method, url, body)
  request.Header = headers
  ...
}
```

This is your decision to either make this change in this PR or the PR that need to send the header into the function.

Reply:
Thanks @gutakk I'll add the change on the PR that needs to test with the header.